guarddog/analyzer/metadata/empty_information.py

```diff
@@ -2,20 +2,23 @@
 
 Detects if a package contains an empty description
 """
-
+from abc import abstractmethod
```

Comment:
The current directory structure has heuristics in a `metadata` folder but also `pypi` and `npm` so I'm confused? Maybe `metadata` is just a leftover and should be removed?

Reply:
in my mind we have `metadata/x.py` -> abstract classes
`metadata/ecosystem/x.py` implementation. wdyt?

Reply:
makes sense